waku/v2/protocol/rln/common.go

```diff
@@ -6,23 +6,23 @@ import (
 	"github.com/waku-org/go-zerokit-rln/rln"
 )
 
-type MessageValidationResult int
+type messageValidationResult int
 
 const (
-	MessageValidationResult_Unknown MessageValidationResult = iota
-	MessageValidationResult_Valid
-	MessageValidationResult_Invalid
-	MessageValidationResult_Spam
+	validationError messageValidationResult = iota
```

Comment:
Should we use all caps for constants to keep them consistent Or is this what is recommended by codeclimate?

Reply:
It's what code climate recommended